specs/README.md

```diff
@@ -19,6 +19,7 @@ Here you'll find specifications for custom elements and other library features.
 | [Progress](../packages/web-components/fast-components/src/progress/progress.spec.md) | :white_check_mark: |
 | [Slider](../packages/web-components/fast-components/src/slider/slider.spec.md) | :white_check_mark: |
 | [Switch](../packages/web-components/fast-components/src/switch/switch.spec.md) | :white_check_mark: |
+| [Tabs](./tabs/tabs.md) | :white_check_mark: |
```

Comment:
Technically the check mark would go in when the component is complete - I plan on pushing a change for the table again today based on some feedback.